python/ray/tests/test_state_api.py

```diff
@@ -3559,6 +3559,28 @@ def test_core_state_api_usage_tags(shutdown_only):
     )
 
 
+# Tests fix for https://github.com/ray-project/ray/issues/44459
+def test_job_info_is_running_task(shutdown_only):
+    ray.init()
+
+    @ray.remote
+    def f():
+        import time
+
+        while True:
+            time.sleep(10000)
+
+    # Create some long running tasks, no need to wait.
+    tasks = [f.remote() for i in range(4)]  # noqa: F841
```

Comment:
any reason why we need 4 tasks here instead of 1?

Reply:
TODO: make it a signal actor enabling task